hawkular-rest-status/pom.xml

```diff
@@ -47,6 +47,11 @@
   </dependencyManagement>
 
   <dependencies>
+    <dependency>
+      <groupId>org.hawkular.commons</groupId>
+      <artifactId>hawkular-tenant-jaxrs-filter</artifactId>
+      <version>0.7.3.Final-SNAPSHOT</version>
```

Comment:
Maven manages this just fine and will update it on releases. Using `${project.version}` used to cause problems when the build is "embedded" into another build (release builds). It has been fixed for some time, but I believe the best practice is still not to use `${project.version}`.